index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
```

Comment:
- [ ] I strongly suggest you avoid using fixed width and height values in your projects unless is really necessary, having these properties with fixed values makes the project looks unresponsive and messy. Please fix this issue.

![image](https://user-images.githubusercontent.com/52635378/164945433-305eda05-f000-40f8-83f1-41be407f8de5.png)

Reply:
the are displaying fine please in my screen and the others reviewer didn't point it out

Reply:
its working perfectly please